util/migrate/table_ops.py

```diff
@@ -11,7 +11,10 @@ def copy_table_contents(source_table, destination_table, conn):
         conn.execute(
             "INSERT INTO `%s` SELECT * FROM `%s` WHERE 1" % (destination_table, source_table)
         )
-        result = list(conn.execute("Select Max(id) from `%s` WHERE 1" % destination_table))[0]
-        if result[0] is not None:
-            new_start_id = result[0] + 1
-            conn.execute("ALTER TABLE `%s` AUTO_INCREMENT = %s" % (destination_table, new_start_id))
+        if conn.engine.name != "sqlite":
```

Comment:
Shouldn't we do some magic around `sqlite_sequence` for sqlite?

Reply:
Reading through the [sqlite docs](https://www.sqlite.org/autoinc.html#:~:text=SQLite%20keeps%20track%20of%20the,an%20AUTOINCREMENT%20column%20is%20created.), I found that the `sqlite_sequence` is automatically updated during the `INSERT INTO ...` stmts while copying contents from one table to another, so the next insert wouldn't need a special handling.